sdk/src/client/api/block_builder/input_selection/requirement/mana.rs

```diff
@@ -287,4 +293,39 @@ impl InputSelection {
                 self.creation_slot,
             )?)
     }
+
+    fn sort_available_inputs_for_mana(&mut self, missing_mana: u64) {
+        // Establish the order in which we want to pick an input
+        let sort_order_type = [
+            BasicOutput::KIND,
+            AccountOutput::KIND,
+            NftOutput::KIND,
+            FoundryOutput::KIND,
+            AnchorOutput::KIND,
```

Comment:
AnchorOutput should be last since we don't support them?

Reply:
If we don't support them at all I can just remove it from the available list

Reply:
Should already be done in `filter_inputs()`